platform/ui/src/contextProviders/DialogProvider.js

```diff
@@ -114,62 +125,84 @@ const DialogProvider = ({ children, service }) => {
     });
   };
 
+  const validCallback = callback => callback && typeof callback === 'function';
+
   return (
-    <DialogContext.Provider value={{ create, dismiss, dismissAll, dialogs }}>
+    <DialogContext.Provider value={{ create, dismiss, dismissAll, noDialogs }}>
```

Comment:
`noDialogs` --> `hasNoDialogs`?

Reply:
Or good ol' `isEmpty`

Reply:
Yeah, i think isEmpty is simpler, updated, thanks.